New python-only modules failing on build step

### What happened?

PR #5279 fixed one aspect of QA tests, but in the process introduced a bug that prevents a new python-only module from building.

See https://github.com/gnuradio/gnuradio/pull/5279#issuecomment-971941003 and below.



### System Information

Ubuntu 20.04
Source Build




### GNU Radio Version

3.9 (maint-3.9)

### Specific Version

3.9.4.0

### Steps to Reproduce the Problem

- Make a new OOT module using `gr_modtool`
- add a python block
- `make` will fail with "No rule to make target" for the pybind "_python" target

### Relevant log output

```shell
make[2]: *** No rule to make target '../python/pyonly_python', needed by 'python/CMakeFiles/copy_module_for_tests'.  Stop.
make[1]: *** [CMakeFiles/Makefile2:321: python/CMakeFiles/copy_module_for_tests.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
```

--- a/gr-utils/modtool/templates/gr-newmod/python/CMakeLists.txt
+++ b/gr-utils/modtool/templates/gr-newmod/python/CMakeLists.txt
@@ -32,13 +32,10 @@ include(GrTest)
 
 set(GR_TEST_TARGET_DEPS gnuradio-howto)
 
-# Create a module directory that tests can import. It includes everything
-# from `python/` and the built bindings shared lib.
+# Create a package directory that tests can import. It includes everything
+# from `python/`.
 add_custom_target(
   copy_module_for_tests ALL
   COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_CURRENT_SOURCE_DIR}
           ${CMAKE_BINARY_DIR}/test_modules/howto/
-  COMMAND
-    ${CMAKE_COMMAND} -E copy_directory ${CMAKE_CURRENT_BINARY_DIR}/bindings/
-    ${CMAKE_BINARY_DIR}/test_modules/howto/
-  DEPENDS howto_python)
+)

--- a/gr-utils/modtool/templates/gr-newmod/python/bindings/CMakeLists.txt
+++ b/gr-utils/modtool/templates/gr-newmod/python/bindings/CMakeLists.txt
@@ -36,4 +36,12 @@ GR_PYBIND_MAKE_OOT(howto
    gr::howto
    "${howto_python_files}")
 
+# copy in bindings .so file for use in QA test module
+add_custom_target(
+  copy_bindings_for_tests ALL
+  COMMAND
+    ${CMAKE_COMMAND} -E copy "${CMAKE_CURRENT_BINARY_DIR}/*.so"
+    ${CMAKE_BINARY_DIR}/test_modules/howto/
+  DEPENDS howto_python)
+
 install(TARGETS howto_python DESTINATION ${GR_PYTHON_DIR}/howto COMPONENT pythonapi)

